client/reader/stream/index.jsx

```diff
@@ -165,9 +165,8 @@ class ReaderStream extends React.Component {
 		}
 	}
 
-	componentWillReceiveProps( nextProps ) {
-		const { streamKey } = nextProps;
-		if ( streamKey !== this.props.streamKey ) {
+	componentDidUpdate( prevProps ) {
+		if ( prevProps.streamKey !== this.props.streamKey ) {
 			this.props.resetCardExpansions();
 			this.props.viewStream( { streamKey } );
```

Comment:
this should have bombed eslint... maybe this needs a rebase to pick up https://github.com/Automattic/wp-calypso/pull/26528